Bioinfo_joint/HW4_modules/natls.py

```diff
@@ -0,0 +1,110 @@
+def dict_comp(nucleotide: str) -> str:

```

Comment:
Мне нравится, что есть функция, которая возвращает комплиментарный нуклеотид по переданному. Это может упростить чтение кода и понимание логики лучше, чем прямой вызов словаря. Но на мой взгляд, в таком случае, стоит назвать функцию таким образом, чтобы она отражала суть действия. Например
```suggestion
def get_compliment(nucleotide: str) -> str:
```
В таком случае, в местах вызова этой функции будет более понятно, с каким значением будут производиться операции